utils/merge_and_push_translations.sh

```diff
@@ -57,7 +57,32 @@ echo "Working tree is clean, continuing"
 # Print all commands.
 set -x
 
-# Pull translations from Transifex
+# Update source catalogs
+python3 utils/buildcat.py
+
+undo_oneliner_diffs() {
+  # Undo one-liner diffs of pure timestamps with no other content
+  set +x
+  for entry in $(git diff --numstat po/ | sed -En 's/^1\t1\t//p'); do
+    if [ -z "$(git diff "$entry" | grep '^[+-][^+-]' | grep -v '^[+-]"POT-Creation-Date:')" ]
+    then
+      echo "Skipping changes to $entry"
+      git checkout $entry
+    fi
+  done
+  set -x
+}
+undo_oneliner_diffs
+
+if [ -z "$(git status -s)" ]; then
+  true # nothing changed, do not upload to transifex
+else
+  # Push source catalogs to Transifex
+  tx push -s
+  sleep 5 # ?? # wait for translation files to be updated
```

Comment:
Transifex should be pretty quick with updating this, but maybe wait one minute to be sure. This workflow doesn't need to be very fast.